scripts/docblock.mjs

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env zx
+
+/*
+ * Copyright 2024 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { parseFileSync } from '@swc/core';
+import { parse } from 'comment-parser';
+import { fs, glob } from 'zx';
+
+const SWC_OPTIONS = {
+  syntax: 'typescript',
+  target: 'es2022',
+};
+
+function exportsReducer(acc, node) {
+  if (node.type === 'ExportDeclaration') {
+    if (node.declaration.identifier) {
+      acc.push(node.declaration.identifier.value);
+    } else if (node.declaration.declarations) {
+      for (const inner of node.declaration.declarations) {
+        acc.push(inner.id.value);
+      }
+    }
+  }
```

Comment:
Type exports are also `declaration.id.value`

Reply:
For the context - docblock - does it matter? Do we want to expect docblock for type alias'?

Reply:
Yea. Still a part of the public facing API. And, depending on how the type is constructed, intellisense won't help. Unless you "goto definition" several times.

Reply:
I think I got this fixed. Let me knwo.